lib/hoge.go

```diff
@@ -1,6 +1,8 @@
 package lib
 
+import "github.com/crifff/private"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/crifff/private (can't find import: )

<!-- Comment by [SideCI](https://sideci.com) -->